styles/styles.css

```
@@ -0,0 +1,340 @@
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("/fonts/ptsans-400.woff2");
```

```suggestion
  src: url("../fonts/ptsans-400.woff2");
```
И для остальных относительных путей в CSS файле.

[2.3. Абсолютные и относительные пути](https://up.htmlacademy.ru/htmlcss/36/book/graphics/links).